app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -201,6 +210,7 @@ class HomeMenu(
             extensionsItem,
             syncSignInMenuItem,
             accountAuthItem,
+            manageAccountAndDevicesItem,
```

Comment:
Just to double check, with the current approach this new menu option would only appear when the user is on the app's homescreen, it would not also appear when browsing (which would need a similar declaration in `DefaultToolbarMenu`). Is this intended?

Reply:
Yes we plan to make minimal changes, so just add this to home screen menu.